README.md

```diff
@@ -1,11 +1,16 @@
 # Chel: Chelonia Command-line Interface
 
-🚧 Under construction! 🚧
+Modern CLI for Chelonia contract development, deployment, and management.
+
+## All Available Commands
 
 ```
 chel
 chel help [command]
 chel version
+chel build
+chel pin <version> <manifest-file-path> [--only-changed | --overwrite]
```

Comment:
- `--only-changed` should be removed here in the readme as well
- The `<version> should come at the end
- The options should be before the mandatory params